CHANGELOG.md

```diff
@@ -386,6 +386,7 @@ All notable changes to this project will be documented in this file.
 [`overflow_check_conditional`]: https://github.com/Manishearth/rust-clippy/wiki#overflow_check_conditional
 [`panic_params`]: https://github.com/Manishearth/rust-clippy/wiki#panic_params
 [`partialeq_ne_impl`]: https://github.com/Manishearth/rust-clippy/wiki#partialeq_ne_impl
+[`possible_missing_comma_in_array_formatting`]: https://github.com/Manishearth/rust-clippy/wiki#possible_missing_comma_in_array_formatting
```

Comment:
I'd call that `possible_missing_comma`, this is probably long enough already :smile: